assets/src/edit-story/components/canvas/layout.js

```diff
@@ -120,13 +120,13 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
     hasHorizontalOverflow ? 'flex-start' : 'center'};
   align-items: ${({ hasVerticalOverflow }) =>
     hasVerticalOverflow ? 'flex-start' : 'center'};
-  overflow: ${({ hideScrollbars }) =>
-    hideScrollbars ? 'hidden' : 'var(--overflow-x) var(--overflow-y)'};
+  overflow: ${({ overflow }) =>
+    overflow ?? 'var(--overflow-x) var(--overflow-y)'};
 
   ${({ isControlled, hasVerticalOverflow, hasHorizontalOverflow }) =>
     isControlled &&
     css`
-      overflow: hidden;
+      overflow: ${({ overflow }) => overflow ?? 'hidden'};
```

Comment:
Hm, couldn't this also come from the `showOverflow`? Or is there a possibility where `showOverflow === false` but `overflow === 'visible'`? Thoughts?

Reply:
Don't ask! 😂 

I've spent too much time on this already 🙈 . It could probably be simplified - or maybe even split into three completely different layers now just using the same grid, as the three layers are actually pretty different by now 🤷 . But I'll leave that as an exercise for the reader...

Reply:
Haha! Mb this can be followed up in another PR 😆 Seems like `showOverflow` means `overflow: visible` and `!showOverflow` means `visibility: hidden` (at least that was the original idea for `showOverflow`) so unless I'm missing something then `overflow` might be duplicating the point of `showOverflow`.